### DISCUSSION TITLE
Procreate Dreams

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#406] [General]
I just finished watching the Procreate presentation, and it looks like they're making a revolutionary animation app, launching November 22nd, for $19.99. They showed a lot of stuff in it, they even got Aaron Blaise, one of the animators from the Disney Renaissance, the co-director of Disney's Brother Bear, the host of The Art of Aaron Blaise on YouTube, to make a short for the presentation. Dude, if this doesn't spice up the competition for the best animation software, I don't know what will. Keiwando, I'd like to know what your thoughts are about this, and what you plan to do for ToonSquid going forward with new competition on the horizon.

**Comment by ota-works** [comment]
oh, I was expecting this conversation!!!

**Reply by Lioncub901** [reply]
Me too.

**Comment by overcesium** [comment]
I've watched the presentation too. My thoughts immediately went to Toonsquid. I wish you all the best Keywando, I'm sure that Toonsquid will still be able to hold up againts the competition. You made the best animation app available on Ipad and I believe you alone will be able to impress us all in the future.

**Comment by VeiloPot** [comment]
In my opinion, i think it would be cool if keiwan can ask the procreate people to collab with him if possible.

**Comment by trinhpxl** [comment]
it looks very promising but its a shame im not a fan of the ui D: hopefully that’ll change in the future idk. but i wish all the best for toonsquid, especially the respect i have for a one man team !! i still look forward to each and every update :D and i cant wait to see more from toonsquid ! if there is going ti be any competition, that would be exciting to see xD

but i dont want to put pressure on keiwan for this, yall do realize its a one man team going against a whole dev team on proc, right ? 😭 i just hope he’ll be okay !

**Reply by Yelisava** [reply]
True

**Comment by VeiloPot** [comment]
Also I usually don't think there would be a rivalry between the two apps, because i remember asking one of the people from procreate about if they know about Toonsquid, they said they're really interested what they would see.

**Comment by Lioncub901** [comment]
There is no Vfx graph. I want one. That is all I care about.

**Reply by ota-works** [reply]
VFX graph? Could you please elaborate on that, please? because there's effects like blur, and some others are available in Dreams

**Comment by dreeze12** [comment]
goodbye toonsquid. thank you..
 welcome procreate dreams.

**Reply by ota-works** [reply]
Not yet, lol, but we should still support Toonsquid. I believe there will be some great updates that will come with this app

**Reply by DinoAwakening** [reply]
Not even in a million years. Procreate Dreams might have their fancy brushes (Which I believe they maybe gonna be supported by ToonSquid in the future), the only difference between Dreams and ToonSquid so far is the lack of Filters/VFX by ToonSquid’s side. Right in the other hand, ToonSquid is the perfect hybrid as a Drawing/Animation Software. It takes the great things and touch that Procreate provides, but almost non of its cons. And think about it… It’s Keiwando soloing animation software development in the entire Apple Ecosystem (For now, because he might want to port it to Android or even Windows at a WAY TOO AFFORDABLE price). Procreate is an entire team, and ToonSquid remains still feeling better. :3

**Reply by ota-works** [reply]
Gonna have to disagree with that, in realistic terms. Procreate is a team, and this is just version 1 that they're releasing. Who knows how thing would be when they get to version 5, for example?
But Toonsquid is nothing to sleep on either. I'm really looking forward to its future updates

**Reply by trinhpxl** [reply]
yeah tbh dreams is barely exciting for me considering toonsquid and how it raised the bar for me 😭 clean ui, easy to understand, godsend of a tweening system compared to say, callipeg or what dreams looked like

**Comment by alx-xlx** [comment]
I wouldn't say toonsquid has lost almost all value with the procreate dream. As Procreate Dreams looks like a frame by frame animation and not vector based.
However toonsquid may face competition from the linearity's motion app scheduled to release this year. Its the same company that released vectornator and changed its name to curve. This Motion app will be having vector based animation, with a free tier system.

Now with these two app launches toonsquid's profit will decrease. Both apps have massive funding behind them. And so yes now you can say that toonsquid will be loosing a lot of value on appstore.
Both these two companies focused on static/2d drawings with their Procreate & Vectornator apps in the beginning and now both have moved into animation. Where as Toonsquid developed - 2d drawing, 2d animation, vector drawing, vector animation. And so wasn't able to perfect any category specifically. 

It was really amazing to see one man vs a team of people. However they have caught on.

**Reply by ota-works** [reply]
Thanks for the heads-up on Linearity's Motion. Never heard of it till now.
I also discovered, upon researching Motion, that IbisPaint is going to have an animation mode, which is great to hear.
I think the best we can do is to still stick by Keiwan and support him by being active and looking forward to Toonsquid's future updates. I mean, we've already got the app and have done some works with them, so what's wrong seeing the potential this app will have in the future?

Could we also update each other on the new animation apps to look forward to or have glanced over?

**Reply by Cat-Butler** [reply]
From what I've seen so far, Linearity looks like it will be good for motion graphics, but not so great for character-style animation.

**Reply by alx-xlx** [reply]
I am a little sour on Linearity and I am distancing my usage from its illustrator app. They recently blocked all users from accessing the files without having to Log in to their Linearity Cloud and forced all users to upload their arts to cloud if they wanted to continue using the art files.
**So NOW basically they have access to all the arts that are being created every day, hour, mins, sec**. That is too much power !!

And their is no way to turn that off. Somehow I have to trust them to not use my art in their upcoming AI features.

These are very **anti-consumer behaviours** and I wonder that other nightmares they have in their basket. So I think its better to not include their apps into my pipeline.

**Reply by kevincreative** [reply]
Yeah. Nope!

**Comment by ota-works** [comment]
Massively geeking out on this, but check this out!!! ProDreams will have non-destructive transformation tools!!!
https://youtu.be/AzPgJNWCVCc?t=492

This link comes with the time it is demonstrated.

I could be wrong, but I really hope I'm not. Fingers crossed!

**Reply by DinoAwakening** [reply]
The warp scaling mode is pretty much the same tho… The only thing different on Dreams is that it acts as a Vector layer, but trust me, ToonSquid has it at least as a Bitmap scaling mode, which is great.

**Reply by ota-works** [reply]
what makes it wrong being a Vector Layer tho?

**Reply by DinoAwakening** [reply]
There’s nothing wrong about it. I’ve never said it was a bad thing, just different :)

**Reply by ota-works** [reply]
oh okay. Thanks for the clarification.
What makes the Bitmap scaling mode so unique. Could you please explain it?

**Reply by DinoAwakening** [reply]
It is a bit more of the same. The only thing that changes is that with Bitmaps it is easier to remove parts of the drawing that are defined by a unit of measurement such as pixels. Whereas vectors are defined by coordinates, so it's like working literally with atoms instead of blocks. :3 

Edit: And trust me, the human eye doesn’t need such a fractal measurement. Which is basically infinite D:

**Comment by Cat-Butler** [comment]
There's room in my toolbox for multiple apps, particularly at their current prices.

ToonSquid, I suspect might still be better for zoomable vector brushes and path shapes and more cut-out style animation rigging (the current hierarchy and switch-style symbols for mouths and hands) with perhaps some frame-by-frame - in some ways, a little like Moho (which has an announcement coming up soon).

Without actually trying it yet, it looks like Procreate Dreams will be great for frame-by-frame work with perhaps a little keyframing, perhaps more like TVPaint. I do like the look of the timeline.

Then there are times when I might bring in other apps, e.g. for special effects, whether it be in a video editor or an animation app.

**Reply by ota-works** [reply]
YESSSSS!!! This comment right here!
We cannot be hung up on only one single animation app. We need to learn to see the benefits in other apps and use them accordingly. Thankfully, Toonsquid, Callipeg and Procreate Dreams are all one-time purchases, so why don't we get them all and support iPad animation development?

**Reply by ota-works** [reply]
It would be very surprising if Moho ported to iPad. It would be very surprising

**Reply by Cat-Butler** [reply]
No, I'm expecting any announcement to be for Moho 14, still for desktop only but with some great new features.
I was merely mentioning some (limited) similarities between ToonSquid and Moho - the use of symbols and frame markers is similar to switch layers, the use of vector shapes, the idea of rigging (still fairly basic in the current version of ToonSquid, but with possibilities even in its current format).

**Reply by ota-works** [reply]
Moho 14 released and it looks AMAZING!!! Really cool new features!

**Comment by Yelisava** [comment]
Yeah, I also thought immediately about Toonsquid. I'm so thankful for the dev for creating the app. You achieved a lot of karma points and definetely will proceed to do so!

**Comment by overcesium** [comment]
I should refrain from posting this, but after testing the beta of Procreate Dreams, I can assure you all that AT THE MOMENT, Toonsquid remains the best options available on Ipad for professional animators. The beta misses a lot of feature that are an absolute necessity for any animation software, maybe they will implement them in the final release,  but I really don't see this new app as a revolution. Yes, it's going to grow, being developed by a big corp, but the stuff you can do right now in Toonsquid are extremely advanced compared to the current state of Dreams. Just wanted to let you know, keep loving Toonsquid because Keywando alone did an outstanding job

**Reply by Cramos821** [reply]
What features are missing?

**Reply by overcesium** [reply]
Lasso and selection tool was missing from the beta, I sure hope they add it in the final release. The "jump to next keyframe/ drawing" function, camera and in general, it doesn't really feel like a professional software. We'll see in a few days if they can change my mind

**Reply by trinhpxl** [reply]
so turns out you were right, lasso tool is not in release. lots of features are missing… it was a big let down for release ngl.
for an app six years in development i expected more from a team at procreate 😔💔  no brush editing system in dreams itself, ui didnt feel as intuitive, reordering drawing layers isnt a feature, etc. it feels really bare bones overall :/

ANYWAYS THOUGH TOONSQUID SOLOS 🗣️‼️‼️‼️‼️‼️

**Comment by FoxeyeStudios** [comment]
I’m really thankful to see there are still many users supporting this man and his brilliant app. It’s very impressive how one guy was able to launch a powerful animation app for IPad faster than an entire team of developers did, regardless of how long it took or who started working first. That being said, I can imagine it’s a bit of a handful working on an app while needing to work and have a life away from the keyboard. I will say I did have some concerns for ToonSquid when Procreate Dreams was announced, though after seeing others test the beta version, I have a feeling ToonSquid is going to be just fine. I see a lot of pros and cons for each app and even made a list of comparison, probably prematurely and some of the things I listed may be inaccurate by the time it’s fully released to the public. I hesitate to make another comparison considering both will continue to grow and improve, therefore making it obsolete. Going by an unbiased opinion, ToonSquid is still preferable than Procreate Dreams. I will continue to support ToonSquid and Keiwando through it all.

**Comment by ota-works** [comment]
Guys, I understand the need to support Toonsquid, as I do as well, but let's not pit iPad animation apps against each other.
We are chartering new grounds here, and the worst we can do is to say one app is superior, rather please let us embrace each app for its positives and negatives too, and realize that we can jump between apps to help with our works.

I'm personally so excited for Procreate Dreams, as it has brushes that I couldn't find in Toonsquid, but that doesn't mean I'm abandoning Toonsquid. 
I love using Toonsquid as a compiler and I don't think that'll change, especially with its Symbols feature.
I'm also embracing Adobe Fresco, free version, because it can help in creating 2D effects through the Particles feature it has.


Tl;dr:  Every app is special in and of its own way, so let's not make a competition out of it and enjoy what animation has to offer for us all on the iPad. 🥳🤩

**Comment by BiteyThing** [comment]
I was looking forward to Dreams due to Procreate's amazing drawing tools... But all you get in Dreams is a brush, smudge, eraser, and color picker for drawing. Not even a shortcut for the eraser.... I guess you're expected to import anything that requires standard drawing tools from Procreate... The only thing ToonSquid has to worry about, if Savage continues down this road, is the crazy hype-train behind Dreams.... It's well worth the $20 but at the same time, it's pretty far from what I was expecting.

Maybe the Dreams hype will open people's eyes to the possibilities of creating animation on the iPad and they will find ToonSquid.

**Comment by kevincreative** [comment]
It all depends on what type of work you do. For me, currently ToonSquid works brilliantly for the stuff I do. Mostly keyframe based, motion graphic, flat vector looking styled work. There is certainly room for different approaches and types of apps. Competition is good. I tried Dreams and to me in its current form of UI and features, it's seems to be geared more toward a frame by frame workflow. What I found a little frustrating was that the timeline kept zooming in and out as I worked and I found myself doing a lot of repetitive finger maneuvering trying to find stuff. It's very different in its workflow, which is not necessarily a bad thing but good to be aware of going in. Very heavily gesture driven, which once conquered I admit would probably make for a quick and intuitive workflow. I guess for the price of both of these apps there's almost no excuse for not getting them both and using each for what they're best at. It's nice to have these choices. :-)

**Comment by eliSe3665** [comment]
Just bought ToonSquid AFTER Procreate Dreams (which shows promise but I wanted a more "mature" app right away) and thanks to a Procreate Dreams video tutorial/review. Now I am rolling up my sleeves to learn how to use this software as well.  TurboSquid and Procreate Dreams can coexist in the marketplace and on each other's iPads. No software is an island... they are tools, and in my toolbox I like to have more choice (also considering the prices). Competition and collaboration makes everyone grow better. Long live ToonSquid!

**Reply by kevincreative** [reply]
I did the same thing. I bought Dreams, played with it for a couple of days then bought ToonSquid. I have since completed a small project in each so far and found that ToonSquid is better suited right now for what I do. That may change but like you say its good to have the choice. ToonSquid is pretty solid though as it sits right now for client work.
